src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -118,7 +118,7 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="InitialJsonParseErrorText" xml:space="preserve">
-    <value>Settings could not be loaded from file. Check for syntax errors, including trailing commas.</value>
+    <value>There was an error when loading the settings. Here is the error message:</value>

```

Comment:
If this is already getting concatenated to the more specific error, I don't think you need the clause about HERE IS MORE ERROR:

```suggestion
    <value>There was an error when loading the settings.</value>

```